### PR TITLE
Fix get version and release on develop branch

### PR DESCRIPTION
**Pull request by magarciasopo**
Changes for getting version and release properly on develop branch

### CODE REVIEW

File: rpm/get_version_string.sh [hunk 1]
```diff
@@ -1,8 +1,19 @@
 #!/bin/bash
+
+# Copyright 2014 Telefonica Investigacion y Desarrollo, S.A.U
+#
 #
-# Bash lib to know the RPM version and revision from a GitHub repository
+# Bash lib to know the RPM version and revision from a Github repository
 # Call method get_rpm_version_string to obtain them for rpmbuild
+# The result appear on the vars ver and rel
+# The requisites are tags similar to 0.1.0/KO. This tag must be created by 'git tag -a 0.1.0'
```
Comment (fgalan) [inline rpm/get_version_string.sh]: 0.1.0/KO -> 0.1.0-KO

File: rpm/get_version_string.sh [hunk 2]
```diff
@@ -123,3 +134,6 @@ is_pdi_compliant()
     *)  echo 1 ;;
    esac
 }
+
+#cd $1
```
Comment (fgalan) [inline rpm/get_version_string.sh]: Lefover to be removed? Or it is included in commented way on purpose?

File: rpm/get_version_string.sh [hunk 3]
```diff
@@ -1,8 +1,19 @@
 #!/bin/bash
+
+# Copyright 2014 Telefonica Investigacion y Desarrollo, S.A.U
```
Comment (fgalan) [inline rpm/get_version_string.sh]: 2014 -> 2015

(Except if you want to consider the date of the original script, that -maybe, I'm not sure- was created in 2014).

File: rpm/get_version_string.sh (hunk 1 above)
Comment (magarciasopo) [inline rpm/get_version_string.sh]: Fixed on commit d3ca18852ac4dddbb99bcbacdaaeac934f7f4e2c

File: rpm/get_version_string.sh (hunk 2 above)
Comment (magarciasopo) [inline rpm/get_version_string.sh]: Fixed on commit d3ca18852ac4dddbb99bcbacdaaeac934f7f4e2c

File: rpm/get_version_string.sh (hunk 3 above)
Comment (magarciasopo) [inline rpm/get_version_string.sh]: Fixed on commit d3ca18852ac4dddbb99bcbacdaaeac934f7f4e2c

**Comment by fgalan** [comment]
LGTM

**Comment by fgalan** [comment]
(I click on close by mistake, sorry. Reopening)

**Pull request merged**